zeppelin-interpreter/pom.xml

```diff
@@ -137,6 +137,12 @@
     </dependency>
 
     <dependency>
+      <groupId>net.java.dev.jna</groupId>
+      <artifactId>jna</artifactId>
+      <version>4.1.0</version>
```

Comment:
Any reason not to use 4.2.2? (http://search.maven.org/#artifactdetails%7Cnet.java.dev.jna%7Cjna%7C4.2.2%7Cjar)